Ntuplizer/test/ntuplize_cfg.py

```diff
@@ -193,16 +202,20 @@
 #############################################################################
 FlowSteps = []
 
-# everybody needs vertex cleaning
-from UWVV.AnalysisTools.templates.VertexCleaning import VertexCleaning
-FlowSteps.append(VertexCleaning)
+# Vertex cleaning for ZZ
+if zz:
+    from UWVV.AnalysisTools.templates.VertexCleaning import VertexCleaning
+    FlowSteps.append(VertexCleaning)
 
 # everybody needs basic lepton stuff
 from UWVV.AnalysisTools.templates.ElectronBaseFlow import ElectronBaseFlow
 FlowSteps.append(ElectronBaseFlow)
 
 from UWVV.AnalysisTools.templates.MuonBaseFlow import MuonBaseFlow
 FlowSteps.append(MuonBaseFlow)
+if zz:
```

Comment:
As with the vertex cleaning, this needs to happen for the ZZ 3l control region too. Does WZ really not do this?

Reply:
Similar point as before, I will turn it off only for ZZ. For the most part I think it's reasonable to include this, but it seems that most analyses don't and it makes syncing a pain, so I leave it off for now.